### PR TITLE
Add program flow item view to workflows

### PR DESCRIPTION
**Pull request by LisaManresa**
- [x] Add the Clubhouse Story ID: [ch21517]

Screen Shot 2019-12-05 at 6 35 57 AM

### CODE REVIEW

File: src/js/views/admin/program/workflows/workflows.scss [hunk 1]
```diff
@@ -43,21 +43,21 @@
...
 
-.workflows__action-icon {
+.workflows__icon {
   position: relative;
   top: -1px;
   vertical-align: 1px;
 
   .icon {
-    color: color(blue);
+    color: $black-60;
```
Comment (paulfalgout) [inline src/js/views/admin/program/workflows/workflows.scss:61]: The action icon is blue.  Only the flow folder is grey
https://marvelapp.com/68ec5id/screen/63607908

File: src/js/views/admin/program/workflows/workflows_views.js [hunk 2]
```diff
@@ -77,6 +73,17 @@ const ItemView = View.extend({
...
   },
+});
+
+const ActionItemView = ItemView.extend({
+  template: ActionItemTemplate,
+  onClick() {
+    Radio.trigger('event-router', 'program:action', this.model.get('_program'), this.model.id);
+  },
+  onRender() {
+    ItemView.prototype.onRender.apply(this);
```
Comment (paulfalgout) [inline src/js/views/admin/program/workflows/workflows_views.js]: I'd rather duplicate the `onRender`s than call the prototype

File: src/js/apps/admin/program/workflows/workflows_app.js [hunk 3]
```diff
@@ -17,12 +17,16 @@ export default App.extend({
...
+      Radio.request('entities', 'fetch:programActions:collection:byProgram', { programId: program.id }),
+      Radio.request('entities', 'fetch:programFlows:collection', { program: program.id }),
+    ];
   },
-  onStart({ program }, actions) {
+  onStart({ program }, [actions], [flows]) {
     this.actions = actions;
 
-    this.showChildView('content', new ListView({ collection: actions }));
+    const collection = new Backbone.Collection([...actions, ...flows]);
```
Comment (paulfalgout) [inline src/js/apps/admin/program/workflows/workflows_app.js]: I think both of these should be `...actions.models` and `..flows.models`  But does this work?  I can't figure out why..

**Review by paulfalgout — CHANGES_REQUESTED** [review]

File: src/js/apps/admin/program/workflows/workflows_app.js (hunk 3 above)
Comment (LisaManresa) [reply src/js/apps/admin/program/workflows/workflows_app.js]: Huh, I'm not sure why it works either. I'll change it to models.

**Comment by LisaManresa** [comment]
I need to address the cypress tests, so I'm going to tag this as do not merge for now.

File: test/integration/admin/programs/program/workflows.js [hunk 4]
```diff
@@ -44,9 +43,18 @@ context('program workflows page', function() {
         fx.data = testAction;
         return fx;
       })
+      .routeProgramFlows(fx => {
```
Comment (paulfalgout) [inline test/integration/admin/programs/program/workflows.js:46]: I think this may be flaky if the random timestamp isn't after the `updated_at`s above

**Review by paulfalgout — CHANGES_REQUESTED** [review]

**Comment by paulfalgout** [comment]
failing tests seem legit

File: test/support/api/program-flows.js [hunk 5]
```diff
@@ -0,0 +1,33 @@
+import _ from 'underscore';
+import { getResource, getRelationship } from 'helpers/json-api';
+
+Cypress.Commands.add('routeProgramFlows', (mutator = _.identity, programId) => {
+  cy
+    .fixture('collections/program-flows').as('fxProgramFlows')
```
Comment (luketlancaster) [inline test/support/api/program-flows.js:6]: don't know if you've caught this yet, but you'll need to add a generator for this collection here: https://github.com/RoundingWell/care-ops-frontend/tree/develop/test/fixtures/config

**Comment by LisaManresa** [comment]
Tests should be passing now. This should also take care of [ch21521].

**Pull request merged**